content/shmem_init_thread.tex

```diff
@@ -0,0 +1,55 @@
+\apisummary{
+Initializes the OpenSHMEM library, similar to \FUNC{shmem\_init}, and performs any
+initialization required for supporting the provided thread level.}
+
+%Initializes the \openshmem{} library, similar to \FUNC{shmem\_init}, and in addition performs the initialization required for thread-safe invocation of \openshmem{} functions.}
+
+\begin{apidefinition}
+
+\begin{Csynopsis}
+int shmem_init_thread(int requested, int *provided);
+\end{Csynopsis}
+
+\begin{apiarguments}
+\apiargument{IN}{requested}  {The thread level support requested by the user.
+}
+\apiargument{OUT}{provided}{The thread level support provided by the \openshmem{} implementation.}
+\end{apiarguments}
+
+
+\apidescription{
+\FUNC{shmem\_init\_thread}  initializes the \openshmem{} library similar to
+\FUNC{shmem\_init}, and in addition performs the initialization required for
```

Comment:
Using "similarly" seems wrong to me, unless I reconstruct the sentence in a different way. 
How about this ?
"shmem_init_thread initializes the OpenSHMEM library similar to shmem_init. In addition, it performs the initialization required for supporting the provided thread level."